baybe/acquisition/acqfs.py

```diff
@@ -120,6 +120,21 @@ def get_integration_points(self, searchspace: SearchSpace) -> pd.DataFrame:
         return result
 
 
+########################################################################################
+### Knowledge Gradient
+@define(frozen=True)
+class qKnowledgeGradient(AcquisitionFunction):
+    """Monte Carlo based knowledge gradient."""
+
+    abbreviation: ClassVar[str] = "qKG"
+
+    num_fantasies: int = field(validator=[instance_of(int), gt(0)], default=128)
+    """Number of fantasies to draw for approximating the knowledge gradient.
+
+    More samples result in a better approximation, at the expense of both memory and
+    wall time."""
```

Comment:
Guess there are some words missing, to be make it precise?
```suggestion
    More samples result in a better approximation, at the expense of both increased
    memory footprint and wall time."""
```

Reply:
btw this line was taken 1to1 from botorch